src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -1062,6 +1133,24 @@ public RunningBackup OnGoingBackup(long taskId)
             return result;
         }
 
+        public void HandleDatabaseValueChanged(string type, object changeState)
+        {
+            if (type != nameof(DelayBackupCommand))
+                return;
+
+            using (_serverStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                var state = (DelayBackupCommand.DelayBackupCommandState)changeState;
+                if (_periodicBackups.TryGetValue(state.TaskId, out var periodicBackup) == false)
+                    throw new InvalidOperationException($"Backup task id: {state.TaskId} doesn't exist");
```

Comment:
what will happen then to the cluster?

Reply:
No problem: we couldn’t locally delay the backup task, which for some reason, does not exist on the current node.
All other nodes with this task will receive a directive to schedule the next backup.